pkgs/games/leela-zero/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, fetchFromGitHub, cmake, boost, eigen, opencl-headers,
+ocl-icd, zlib, openblas }:
+
+stdenv.mkDerivation rec {
+  name = "leela-zero-${version}";
+  version = "0.16";
+
+  src = fetchFromGitHub {
+    owner = "gcp";
+    repo = "leela-zero";
+    rev = "4fd6e694ead45b7cbb14ea76c5bdfcbfa662304b";
+    sha256 = "12v6ngyd51qw0v8cwpwxwrk6hczha5abr9zcwipy8sgfmvv11blm";
+  };
+
+  buildInputs = [ boost eigen opencl-headers ocl-icd zlib openblas ];
+
+  enableParallelBuilding = true;
+
+  sourceRoot = "source/src";
```

Comment:
The GitHub repo includes a CMakeLists.txt, which seems to know how to search dependencies and install the results.

I think it would be better to add `nativeBuildInputs = [ cmake ];` and drop `sourceRoot`, `CXXFLAGS`, and `installPhase`.

Reply:
Ah, very cool.